overlord/devicestate/firstboot.go

```diff
@@ -280,6 +281,12 @@ func populateStateFromSeedImpl(st *state.State, opts *populateStateFromSeedOptio
 		endTs.AddTask(preseedDoneTask)
 		markSeeded.WaitFor(preseedDoneTask)
 	}
+	whatSeeds := &seededSystem{
+		System:  sysLabel,
+		Model:   model.Model(),
+		BrandID: model.BrandID(),
```

Comment:
let's add model.Revision() (as "revision")  and model.Timestamp() (as "timestamp")  as well, for better tracking

Reply:
This is done now